.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{md,markdown}]
+trim_trailing_whitespace = false
+
+[test/mocks/*.txt]
+indent_size = 3
```

Comment:
This seems unnecessary. Don't particularly care how these files are formatted.

Reply:
OK, done.